### DISCUSSION TITLE
Expected Examples

### DISCUSSION THREAD

**Discussion by Zedonboy** [strapi/strapi#7771] [Legacy Topics]
Hi devs.

Am currently looking for a Documentation to help us configure sessionStorage type example using Memmory or Redis.

And how to setup Redis in Strapi.

In fact how to use strapi packages e.g strapi-hook-redis, strapi-views.

Thank you

**Comment by derrickmehaffy** [comment]
The two packages you mentioned `strapi-hook-redis` and `strapi-views` are legacy packages that are not currently being updated and are not guaranteed to work with any of the stable releases.

We don't currently have any documentation for session storage at this moment.

**Comment by MattieBelt** [comment]
Worth mentioning, there is a community middleware for caching. Lately the support for `redis` was added.
Check out [strapi-middleware-cache](https://www.npmjs.com/package/strapi-middleware-cache).

Because this an community middleware you will not find any official documentation about it and for help or occurring errors, you best can contact the creators of the package.
